.gitignore

```diff
@@ -9,3 +9,7 @@ ssl
 Cargo.toml
 target
 Cargo.lock
+target
+Cargo.toml
```

Comment:
Why are there two copies of `target` and `Cargo.toml` here now?

Reply:
Fixed, will push soon